python/mpt/src/mpt/state_diff.py

```diff
@@ -0,0 +1,243 @@
+import logging
+from typing import Dict, Optional
+
+from ethereum.cancun.fork_types import EMPTY_ACCOUNT, Account, Address
+from ethereum.cancun.state import State, get_account, get_storage
+from ethereum_types.bytes import Bytes32
+from ethereum_types.numeric import U256
+
+# Set up logger
+logger = logging.getLogger()
```

Comment:
missing logger config

Reply:
i thought logger config is only written once in the 'main' of a script?

Reply:
checked with grok and it's not necessary to redo the config indeed, however at least `getLogger(__name__)` to have the module name in the message